src/RaceChrono.cpp

```diff
@@ -1,9 +1,14 @@
 #include "RaceChrono.h"
 
+#define MAIN_SERVICE_NAME           (uint16_t) 0x1ff8
```

Comment:
For constants, please use `const`, like here: https://github.com/timurrrr/RaceChronoDiyBleDevice/blob/4433aa6bd9944b815d9db2378dbf8530ebbd65a4/config.h#L11

That is a higher-level way to define constants, and helps avoid a bunch of problems.